install/0000_30_machine-api-operator_02_machine.crd.yaml

```diff
@@ -7,23 +7,23 @@ metadata:
   name: machines.machine.openshift.io
 spec:
   additionalPrinterColumns:
-  - JSONPath: .status.providerStatus.instanceId
+  - JSONPath: .metadata.annotations['machine\.openshift\.io/instance-id']
     description: Instance ID of machine created in AWS
```

Comment:
I think we also need to remove all references to AWS

Reply:
Thanks, fixed.